include/bluetooth/iso.h

```diff
@@ -84,7 +97,10 @@ struct bt_iso_chan_qos {
 	uint8_t				dir;
 	/** Channel interval */
 	uint32_t			interval;
-	/** Channel SCA */
+	/** @brief Channel slave clock accuracy
+	 *
+	 * If unknown for the slaves, this should be set to BT_ISO_SCA_UNKNOWN
```

Comment:
slave is an LL term, do we have a host term that we can use instead? Peripherals?

Is this really set by the application? Should come from the clock driver I think.

Reply:
The parameter in the core spec is called `Slaves_Clock_Accuracy`. It is set by the application/host for the LE Set CIG Parameters command

Reply:
The LL/HCI layer uses the roles master/slave. The GAP/HOST layer uses the roles central/peripheral.
The host terminology is also better in terms of inclusive language, which is in the code of conducts now.

Reply:
Right. 

I checked over some of the other audio/ISO related specs, and CIS central/CIS peripheral does indeed seem to be defined terms now. I'll update it.